src/main/util.ts

```diff
@@ -414,6 +414,12 @@ const runSizeMonitor = async (storageDir: string, maxStorageGB: number): Promise
         return false;
     }
 
+    const minEncounterDuration = getNumberConfigSafe(cfg, 'min-encounter-duration');
+
+    if ((!minEncounterDuration) || (minEncounterDuration < 0) || (minEncounterDuration > 10000)) {
```

Comment:
Totally fine for now, but we're building on a quick fix I made to avoid invalid config - it's a crap user experience to get told "config invalid" and not get told why. Someone should come fix this one day!

Reply:
Agreed - this would really be better served by actually specifying a schema (including defaults) for the cfg store.

Reply:
I'll take a crack at this in a separate PR since it involves some broader changes.